build.assets/tooling/cmd/difftest/README.md

```diff
@@ -0,0 +1,24 @@
+# difftest
+
+This tool finds tests which were changed since a previous git revision.
+
+## Usage
+
+To see changes:
+
+```difftest diff --path . --branch master```
```

Comment:
Nit: I think the correct Markdown syntax for code block is to have triple-backticks on new lines.

Same below. I'd leave a code suggestion but Github's suggestion tool gets confused with these backticks :)

Reply:
Well, ```both``` `options` work ) But you're right, newlines are preferable in this context.